integration/tests/cook/test_basic.py

```diff
@@ -3226,6 +3226,25 @@ def test_kubernetes_checkpointing(self):
         finally:
             util.kill_jobs(self.cook_url, [job_uuid_disabled, job_uuid_enabled])
 
+    @unittest.skipUnless(util.using_kubernetes(), 'Test requires kubernetes')
+    def test_kubernetes_metadata_env_vars(self):
+        docker_image = util.docker_image()
+        container = {'type': 'docker',
+                     'docker': {'image': docker_image}}
+        default_pool = util.default_submit_pool() or util.default_pool(self.cook_url)
+        # info = util.scheduler_info(self.cook_url)
+        try:
+            # need to figure out how to match up COOK_SCHEDULER_REST_URL
+            # leader url and COOK_SCHEDULER_REST_URL can have different schemes right now
+            #command = 'bash -c \'if [[ "${COOK_POOL:-zzz}" == "' + default_pool + '" ]] && [[ "${COOK_SCHEDULER_REST_URL:-zzz}" == "' + info['leader-url'] + '" ]]; then exit 0; else exit 1; fi\''
```

Comment:
Remove commented out lines?

Reply:
I have them as a TODO

Reply:
Should we compare everything except the scheme?

Reply:
it would be scheme and port.

Reply:
OK. I have a slight preference for removing the commented out code lines, but if you feel strongly that it's better with them there, that's fine.